lib/commands/stress.js

```diff
@@ -0,0 +1,66 @@
+const { getEnv, getLogger } = require("../command-helpers");
+const BBB = require("../BBB");
+const _ = require("lodash/fp");
+const stressTest = require("../stress-test");
+
+module.exports = {
+  name: "stress [meeting] [options]",
+  description: "start the stress test",
+  options: (yargs) => {
+    yargs
+      .option("webcams", {
+        alias: "w",
+        type: "number",
+        description: "Number of clients to connect with an active webcam",
+      })
+      .positional("meeting", {
+        description: "The meeting ID",
+        type: "string",
+      })
+      .option("microphones", {
+        alias: "m",
+        type: "number",
+        description: "Number of clients to connect with an active microphone",
+      })
+      .option("listening", {
+        alias: "l",
+        type: "number",
+        description: "Number of clients to connect in listen-only mode",
+      })
+      .option("duration", {
+        alias: "d",
+        type: "number",
+        description:
+          "Duration of the stress test in seconds (after all clients are connected)",
+      })
+      .default("meeting", getEnv("BBB_MEETING_ID"))
+      .default("d", Number(getEnv("BBB_TEST_DURATION", 60)))
+      .default("w", Number(getEnv("BBB_CLIENTS_WEBCAM", 1)))
+      .default("m", Number(getEnv("BBB_CLIENTS_MIC", 1)))
+      .default("l", Number(getEnv("BBB_CLIENTS_LISTEN_ONLY", 2)))
```

Comment:
Shouldn't default values get stored in variables like `DEFAULT_BBB_CLIENTS_LISTEN_ONLY`?